### DISCUSSION TITLE
"Connected Waiting for image" - Android POS devices

### DISCUSSION THREAD

**Discussion by KenyBDG** [rustdesk/rustdesk#10331] [General]
I am a happy RustDesk user. Overall, it’s a great alternative to TeamViewer or AnyDesk. Unfortunately, I am unable to connect to devices running Android. After connecting, the message "waiting for image" appears, and nothing helps—restarts, closing the app, nothing works.
I’m trying to find a solution, but so far without success.

**Comment by 21pages** [comment]
Could you provide the log files on android? https://github.com/rustdesk/rustdesk/wiki/FAQ#access-logs
Remove private informations before upload.

**Comment by KenyBDG** [comment]
[rs_rCURRENT.log](https://github.com/user-attachments/files/18218262/rs_rCURRENT.log)

**Comment by 21pages** [comment]
Could you try out the latest nightly on Android? https://github.com/rustdesk/rustdesk/releases/tag/nightly

If the latest nightly not works, could you try https://github.com/21pages/rustdesk/releases/tag/directx on both side and provide the logs of both sides?

**Comment by KenyBDG** [comment]
Unfortunately,  I am not able to install latest release, because it is a remote location.

**Reply by 21pages** [reply]
Ok, could you try out https://github.com/21pages/rustdesk/releases/tag/directx on the local side and provide the log of the local side?

**Comment by KenyBDG** [comment]
[RustDesk_rCURRENT.log](https://github.com/user-attachments/files/18218396/RustDesk_rCURRENT.log)

**Reply by 21pages** [reply]
Thank you for your feedback. The problem is on the controlled side and have to wait for the update.

**Reply by KenyBDG** [reply]
do I need create any ticket ?

**Reply by 21pages** [reply]
No, you can write feedback here when your Android is available.

**Comment by Aqua1ung** [comment]
Looks like this is a very resilient issue. Currently there is no known fix ... other than making sure your Android phone never goes to sleep, that is.
